modules/core/src/main/java/org/apache/ignite/internal/processors/odbc/ClientListenerAsyncResponse.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.odbc;
+
+import org.apache.ignite.internal.IgniteInternalFuture;
+import org.apache.ignite.internal.processors.platform.client.ClientResponse;
+
+/**
+ * Client listener async response.
+ */
+public class ClientListenerAsyncResponse extends ClientResponse {
+    /** Future for response. */
+    private final IgniteInternalFuture<? extends ClientListenerResponse> fut;
+
+    /**
+     * Constructs async response.
+     */
+    public ClientListenerAsyncResponse(IgniteInternalFuture<? extends ClientListenerResponse> fut) {
+        super(STATUS_SUCCESS, null);
```

Comment:
This calls `ClientResponse(long reqId, String err)` and passes status as request id.

Reply:
Fixed